cacheServer/PullData.bash

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
possibly do in docker file instead of bash? - Auberon
answer: stay in bash. not docker here b/c it makes it modular